### PR TITLE
Add deppy-dev slack channel

### PR DESCRIPTION
**Pull request by asmacdo**
An on cluster general purpose constraint/SAT solver for Kubernetes dependencies. This project is a part of the Operator Lifecycle Manager subproject of the Operator Framework

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -84,6 +84,7 @@ channels:
   - name: csi-secrets-store #this channel is for secrets-store-csi-driver which is a subproject of sig-auth
   - name: de-events
   - name: de-users
+  - name: deppy
```
[the remark this answers is not in the document]
Comment (tylerslaton) [reply communication/slack-config/channels.yaml:87]: +1 to this

[the remark this answers is not in the document]
Comment (asmacdo) [reply communication/slack-config/channels.yaml]: updated ty!

**Comment by jberkus** [comment]
Is this channel for users or for the contributors to deppy?

**Comment by oceanc80** [comment]
> Is this channel for users or for the contributors to deppy?

@jberkus This channel would be for contributors to Deppy and is analogous to the [rukpak-dev](https://kubernetes.slack.com/archives/C038B7MF75M) channel. We have contributors there who are already interested in Deppy and have participated in design discussions but for ease and clarity of communication we'd like to now split out that communication into its own channel.

**Comment by jberkus** [comment]
If it's developers, should it be called deppy-dev?

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Pull request merged**